src/NuGet.Clients/NuGet.PackageManagement.UI/Options/PackageItem.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel.Design;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Collections;
+using NuGet.VisualStudio.Internal.Contracts;
+
+namespace NuGet.Options
+{
+    public class PackageItem
+    {
+        public string ID { get; set; }
+        public ObservableCollection<PackageSourceContextInfo> Sources { get; private set; }
+
+        public string SourcesString
+        {
+            get
+            {
+                string sourcesString = "";
+                for (int i = 0; i < Sources.Count; i++)
+                {
+                    sourcesString += Sources[i].Name;
+                    if (i < Sources.Count - 1)
+                    {
+                        sourcesString += ", ";
+                    }
+                }
+                return sourcesString;
+            }
+        }
+        /// <summary>
+        /// /private string ID;
+        /// </summary>
+        //private ObservableCollection<PackageSourceContextInfo> Sources;
+
+        public PackageItem(string packageid, ObservableCollection<PackageSourceContextInfo> packageSources)
+        {
+            ID = packageid;
+            Sources = packageSources;
+        }
+        public string GetID()
+        {
+            return ID;
+        }
+
+        public ObservableCollection<PackageSourceContextInfo> GetSources()
+        {
+            return Sources;
+        }
+
+        public override string ToString()
```

Comment:
ToString method was made for testing purposes. The Accessibilty screen reader uses it, though, so I kept it

Reply:
In my test with Narrator, I didn't find this string being used. 
Let's double-check that's the case and try to remove this, if possible.

Reply:
Okay I removed the toString method (plus other unused methods)